packages/plans-grid/src/plans-table/style.scss

```diff
@@ -3,6 +3,9 @@
 @import '~@automattic/typography/styles/variables';
 @import '../variables.scss';
 
+/* stylelint-disable scales/font-size */
+// @TODO: re-enable lint rule in https://github.com/Automattic/wp-calypso/issues/44933
```

Comment:
What do you think about adding this just above the specific rule? These floating disable rules tend to get lost over time otherwise.

Reply:
There are more than a few lines that are not passing stylelint. We now have the issue that I've added in the "Now" column of Gutenboarding project.